sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -60,8 +90,22 @@ def __init__(
         self.glossaries = kwargs.get("glossaries", None)
         self.storage_source = kwargs.get("storage_source", None)
 
+    def _to_generated(self):
+        return _TargetInput(
+            target_url=self.target_url,
+            category=self.category_id,
+            language=self.language,
+            storage_source=self.storage_source,
+            glossaries=TranslationGlossary._to_generated_list(self.glossaries)  # pylint: disable=protected-access
```

Comment:
For the case when the user does not pass glossaries

```suggestion
            glossaries=TranslationGlossary._to_generated_list(self.glossaries) if self.glossaries else None  # pylint: disable=protected-access
```